### PR TITLE
[pt-br] Adding Controlling Acess page translation

### PR DESCRIPTION
**Pull request by brendavsantos**
This pull request add the translation of the [Controlling Access to the Kubernetes API](https://kubernetes.io/docs/concepts/security/controlling-access/) page from english to portuguese, it's a contribution that aims to universalize the documentation for everyone in the community.

### CODE REVIEW

File: content/pt-br/docs/concepts/security/controlling-access.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      - IP padrão é a primeira interface de rede não local, mude com a flag `--bind-address`.
+      - solicitação tratada por módulos de autenticação e autorização.
+      - solicitação tratada pelo(s) módulo(s) de controle de admissão.
+      - módulos de autenticação e autorização são executados
+
+## {{% heading "whatsnext" %}}
+
+Leia mais documentação sobre autenticação, autorização e controle de acesso à API:
+
+- [Autenticação](/docs/reference/access-authn-authz/authentication/)
```
Comment (Sea-n) [inline content/pt-br/docs/concepts/security/controlling-access.md:136]: The document links have to change to `pt-br` version if available.

```suggestion
- [Autenticação](/pt-br/docs/reference/access-authn-authz/authentication/)
```

File: content/pt-br/docs/concepts/security/controlling-access.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
+---
+title: Controlling Access to the Kubernetes API
```
Comment (Sea-n) [inline content/pt-br/docs/concepts/security/controlling-access.md:2]: You may want to translate page title as well.

**Comment by Sea-n** [comment]
Hey @brendavsantos,

It's been a while since last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by divya-mohan0209** [comment]
Hey @brendavsantos , since there have been no updates on this PR for a while I shall be closing it. Please reopen it when you feel comfortable making changes.
/close

**Pull request closed without merging**